ArcadiaParties.API/Startup.cs

```diff
@@ -38,6 +39,14 @@ public void ConfigureServices(IServiceCollection services)
             // Temporary use SQLite connection until set connection to Arcadia API
             services.AddDbContext<DataContext>(x => x.UseSqlite(Configuration.GetConnectionString("DefaultConnection")));
 
+            services
+                .AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+                .AddJwtBearer(options =>
+                {
+                    options.Audience = "a2ccb221-60e2-47b8-b28c-bf88a59f7f4a";
```

Comment:
All authentication parameters should be moved to application settings.

Reply:
Done.